api/tests/certification/enrolment/unit/domain/models/Center_test.js

```diff
@@ -1,10 +1,65 @@
 import { Center } from '../../../../../../src/certification/enrolment/domain/models/Center.js';
 import { CenterTypes } from '../../../../../../src/certification/enrolment/domain/models/CenterTypes.js';
 import { CERTIFICATION_FEATURES } from '../../../../../../src/certification/shared/domain/constants.js';
+import { EntityValidationError } from '../../../../../../src/shared/domain/errors.js';
 import { catchErrSync, domainBuilder, expect } from '../../../../../test-helper.js';
 
 describe('Unit | Certification | Session | Domain | Models | Center', function () {
-  describe('#hasBillingMode', function () {
+  context('should verify center type', function () {
+    it('should return an error', function () {
+      // given
+      const notACenterType = 'Not a valid type';
+
+      // when
+      const error = catchErrSync(domainBuilder.certification.enrolment.buildCenter)({
+        type: notACenterType,
+      });
+
+      // then
+      expect(error).to.be.instanceOf(EntityValidationError);
+      expect(error.invalidAttributes).to.deep.equal([
+        {
+          attribute: 'type',
+          message: '"type" must be one of [SUP, SCO, PRO]',
+        },
+      ]);
+    });
+  });
+
+  context('should verify the center habilitations', function () {
+    it('should return an error', function () {
+      // given
+      const notAHabilitationType = { iAM: 'not an habilitation' };
+
+      // when
+      const error = catchErrSync(domainBuilder.certification.enrolment.buildCenter)({
+        habilitations: [notAHabilitationType],
+      });
+
+      // then
+      expect(error).to.be.instanceOf(EntityValidationError);
+      expect(error.invalidAttributes).to.deep.equal([
+        {
+          attribute: 0,
+          message: '"habilitations[0]" must be an instance of "Habilitation"',
+        },
+      ]);
+    });
+  });
+
+  it('should return true when center is a complementary certification pilot', function () {
```

Comment:
On a besoin du contexte et peut-etre aussi de l'autre cas (ca pourrait renvoyer toujours `true`)
```suggestion
  context('#isComplementaryAlonePilot', function () {
    it('should return true when center is a complementary certification pilot', function () {

Reply:
Done